parser/parser.go

```diff
@@ -5,7 +5,7 @@
 package parser
 
 import (
-	"fmt"
+	"github.com/benhoyt/goawk/internal/resolver"
```

Comment:
Let's move this down into a separate block (with a blank line in between stdlib imports and project imports).

Reply:
Sure. Strange, that gofmt doesn't take care of this.